README.md

```diff
@@ -354,7 +354,20 @@ This plugin will also import product associations. It's a zero configuration imp
 
 ### Launch data import from CLI
 
-To actually import data you must first create queue items with the **enqueue command** and then you can import them with the **consume command**.
+To actually import data you must first create queue items with the **enqueue command** or from the **Schedule Akeneo PIM import** button in the product pages. Then you can import them with the **consume command**.
+
+#### Schedule Akeneo PIM import button
+
+This button allows you to queue a product directly from the admin index page. Sylius Akeneo Plugin already override the product resource grid **sylius_admin_product** for you by adding the button for every item.
```

Comment:
I'd not mention the UI import in the `Launch data import from CLI` paragraph. I'd create a new paragraph. I'd also hide the implementation details, I'd describe only how the admin user could import a single product from the admin UI. I'd also add one or two screenshots about that.